lib/stellar/client.rb

```diff
@@ -58,10 +58,25 @@ def initialize(options)
 
     Contract Stellar::Account => Any
     def account_info(account)
-      account_id  = account.address
+      account_id = account.address
       @horizon.account(account_id:account_id)._get
     end
 
+    Contract Stellar::Account => nil
+    def load_account_signers(account)
+      info = account_info(account)
+      account.thresholds = info.thresholds
+      account.signers = Array.new
+      info.signers.each do |signer|
+        account.signers.push(
+          Stellar::AccountSigner.new(
+            signer['key'], 
+            signer['weight'].to_i
+          )
+        )
```

Comment:
You can use `map` to create and push to an array without needing to instantiate them. e.g.:
```suggestion
      account.signers = info.signers.map do |signer|
        Stellar::AccountSigner.new(
          signer['key'], 
          signer['weight'].to_i
        )
```